packages/components/src/molecules/SearchProducts/SearchProductItemControl.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { forwardRef, HTMLAttributes } from 'react'
```

Comment:
**todo**: Instead of using `React.useCallback`, `React.useState`, can you please import them directly from 'react' and use them, just to maintain the consistency in the codebase. Thanks